.github/workflows/ci.yml

```diff
@@ -4,9 +4,7 @@ name: CI
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    branches: [master]
-  pull_request:
-    branches: [master]
+    branches: [Incompressible-taper-channel-flow]
```

Comment:
The CI from the main branch should not be changed.

Reply:
I mean the entire ci.yml file need to be changed back.

Reply:
Sorry I changed the file to test my code for linux build. I have now changed the Ci.yml file back to the original.